src/com/theironyard/novauc/Main.java

```diff
@@ -1,8 +1,169 @@
+
+                    } else {
+                        c.put("name", user.name);
+                        c.put("movie", yamama);
+                        return new ModelAndView(c, "home.html");
+                    }
+
+
+                }),
+                new MustacheTemplateEngine()
+
+
+        );
+
+        Spark.post("/create-account", ((request, response) -> {
+                    String name = request.queryParams("accountName");
+                    String password = request.queryParams("passWord");
+                    User user = users.get(name);
+
+                    if (user == null) {
+                        user = new User(1, name, password);
+                        users.put(name, user);
+
+                    }
+                    if (password.equalsIgnoreCase(user.password)) {
+
+                    }
+                    Session session = request.session();
+                    session.attribute("userName", name);
+                    response.redirect("/");
+                    return "";
+
+                })
+
+
+        );
+
+        Spark.post("/create-movie", ((request, response) -> {
+                    Session session = request.session();
+                    String name = session.attribute("userName");
+                    User user = users.get(name);
+
+                    if (user == null) {
+                        throw new Exception("User is not logged in");
+                    }
+                    String id = request.queryParams("id");
+                    String movieName = request.queryParams("movieName");
+                    String actorName = request.queryParams("actorName");
+                    String movieGenre = request.queryParams("movieGenre");
+                    String movieRating = request.queryParams("movieRating");
+                    Movie movie = new Movie(movieName, actorName, movieGenre, movieRating);
+
+                    user.movies.add(movie);
+
+                    response.redirect("/");
+                    return "";
+
+
+                })
+
+        );
+
+
+        Spark.post("/edit-movie", ((request, response) -> {
+                    Session session = request.session();
+                    String name = session.attribute("userName");
+                    User user = users.get(name);
+                    String updateMovie = request.queryParams("updateMovie");
+                    int upMovie = Integer.valueOf(request.queryParams("upMovie"));
+                    Movie addMovie = new Movie();
+                    for (int i = 0; i < user.movies.size(); i++) {
+                        if (user.movies.get(i).getId() == upMovie) {
+                            addMovie = user.movies.get(i);
+                        }
+                    }
+                    addMovie.setActor(updateMovie);
+                    addMovie.setName(updateMovie);
+                    addMovie.setGenre(updateMovie);
+                    addMovie.setRating(updateMovie);
+
```

Comment:
This is not correct. This will set all of those categories to whatever is put in the box that corresponds to updateMovie.  These should all be different. Please fix this, then resubmit.